### PR TITLE
Enhanced AI & Robotics stream (Robot Week)

### PR DESCRIPTION
**Pull request by sumeetadur**
Preparation for adding custom fragments and stickies components. Uses the "robotWeek" flag.

screen shot 2016-04-20 at 17 38 54

### CODE REVIEW

**Comment by sumeetadur** [comment]
@matthew-andrews / @i-like-robots could you have a look through this?

File: server/transforms/custom-fragments.js [hunk 1]
```diff
@@ -22,10 +22,13 @@ export default () => {
...
 		}
 
-		post(res) {
+		post(res, flags) {
 			if (!this.fragments) return;
 
-			for (const { index, html, bleed, title } of this.fragments) {
+			for (const { flag, index, html, bleed, title } of this.fragments) {
+				
+				if(flag && !flags[flag]) return;
```
Comment (andygnewman) [inline server/transforms/custom-fragments.js]: should make robust incase flags down

```
if(flag && (!flags || !flags[flag])) return;
```

Comment (matthew-andrews) [inline server/transforms/custom-fragments.js]: if flags are down apps won't start so quite safe to do

`if (flags[flag]) {` 😄

File: server/transforms/custom-fragments.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import CustomFragmentsPoller from '../lib/custom-fragments-poller';
 import logger from '@financial-times/n-logger';
 
 const poller = new CustomFragmentsPoller(
-	'https://bertha.ig.ft.com/view/publish/gss/13N3rLTt_mcGJQA2aXEXgeVd0J0JIeqoarR2CGnIdnio/manifest'
+	process.env.CUSTOM_FRAGMENTS_MANIFEST || 'https://bertha.ig.ft.com/view/publish/gss/13N3rLTt_mcGJQA2aXEXgeVd0J0JIeqoarR2CGnIdnio/manifest'
```
Comment (andygnewman) [inline server/transforms/custom-fragments.js]: Is the intention to put the berth a value into config vars for production?

**Pull request merged**